### PR TITLE
Xmc4500 dts updates

### PR DESCRIPTION
**Pull request by talih0**
Rename a file and remove entry from dts.

### CODE REVIEW

File: boards/arm/xmc45_relax_kit/xmc45_relax_kit.dts [hunk 1]
```diff
@@ -27,13 +27,6 @@
 	clock-frequency = <120000000>;
 };
 
-&usic1ch0 {
```
Comment (parthitce) [inline boards/arm/xmc45_relax_kit/xmc45_relax_kit.dts:30]: IMO it makes sense to disable the node here without renaming the soc dtsi

Comment (talih0) [reply boards/arm/xmc45_relax_kit/xmc45_relax_kit.dts:30]: Hi @parthitce, thanks for reviewing.
The usic1ch0 node is already initially disabled in xmc4xxx.dtsi. Do you want to keep the settings and just set it as disabled?

Comment (talih0) [reply boards/arm/xmc45_relax_kit/xmc45_relax_kit.dts:30]: > IMO it makes sense to disable the node here without renaming the soc dtsi

Are you against renaming the file from xmc4500.dtsi to xmc4500_F100x1024.dtsi? Different versions of the XMC4500_xx have different flash and sram sizes. I don't see a clean way to have this without different xmc4500_AxB.dtsi files for each one.
Screenshot_2022-07-03_09-19-44

**Comment by carlescufi** [comment]
@parthitce ping

**Review by parthitce — APPROVED** [review]
LGTM

**Review by carlescufi — APPROVED** [review]

**Pull request merged**